### DISCUSSION TITLE
Shortcut for deleting an instance

### DISCUSSION THREAD

**Discussion by gudipatiR** [talmolab/sleap#1342] [Ideas]
Hi,

I have been using sleap recently and I think a great addition would be a keyboard shortcut for deleting a selected instance. Please let me know how to do this if it already exists.

**Comment by roomrys** [comment]
Hi @gudipatiR,

Thanks for suggesting! 

SLEAP already has this functionality: select the instance and try `Ctrl` + `Backspace`.

Would love to hear more ideas,
Liezl

**Reply by gudipatiR** [reply]
Hi,
What is the mac shortcut?

Thank you

**Reply by gudipatiR** [reply]
also is there a shortcut for undoing something

**Reply by roomrys** [reply]
Hi @gudipatiR,

It should be `cmd` + `delete`. SLEAP also has most keyboard shortcuts documented in Help>Keyboard Shortcuts (in the SLEAP GUI). 

No undo unfortunately 😞

Thanks,
Liezl
